pkg/util/kubernetes/autoscalers/processor.go

```diff
@@ -12,18 +12,24 @@ import (
 	"math"
 	"sort"
 	"strings"
+	"sync"
 	"time"
 
-	datadog "gopkg.in/zorkian/go-datadog-api.v2"
+	"gopkg.in/zorkian/go-datadog-api.v2"
 	autoscalingv2 "k8s.io/api/autoscaling/v2beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	utilserror "k8s.io/apimachinery/pkg/util/errors"
 
 	"github.com/DataDog/datadog-agent/pkg/clusteragent/custommetrics"
 	"github.com/DataDog/datadog-agent/pkg/config"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 	"github.com/DataDog/watermarkpodautoscaler/pkg/apis/datadoghq/v1alpha1"
 )
 
+const (
+	chunkSize = 45
```

Comment:
It may be good to add a comment about this value.

Reply:
I don't want to expose server side mechanisms too much in the agent's code. But I can add a note